### ISSUE TITLE
Error installing Adapters

### ISSUE THREAD

**Issue by mymanga** [apioo/fusio#21]
I am trying to install the SOAP and HTTP adapters. But it try to run;

```
php bin/fusio system:register Fusio\Adapter\Soap\Adapter
php bin/fusio system:register Fusio\Adapter\Http\Adapter
```

I get the errors;
`Provided adapter class does not exist`

The composer installation went fine but still.. :( . I am running

```
Server : nginx/1.4.6 
PHP : php5-fpm 5.6.23
```

**Comment by k42b3** [comment]
Hi, so there was a git tag missing for the SOAP adapter. I have added a first 0.1.0 version so you should be  able to install the SOAP adapter through composer `composer require fusio/adapter-soap` and then register the class at Fusio with `php bin/fusio system:register Fusio\Adapter\Soap\Adapter`. The HTTP adapter gets already installed on installation so you dont need to install it.

**Comment by k42b3** [comment]
This is fixed in the latest 0.3 release
